### PR TITLE
Hom work6

### PR DESCRIPTION
**Pull request by NEULOVIM**

### CODE REVIEW

File: src/ADDlesson6/Computer.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package ADDlesson6;
+
+public class Computer {
+    int price;
```
Comment (DHrytsuk) [inline src/ADDlesson6/Computer.java]: Access modifiers

File: src/lesson6/CreditCard.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        this.balance = balance;
+    }
+
+
+    public void plusSum(double sum) {
+        balance += sum;
+
+    }
+
+    public void minusSum(double sum) {
```
Comment (DHrytsuk) [inline src/lesson6/CreditCard.java]: Ideallym this method should return remaining balance

File: src/lesson6/CreditCard.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        balance -= sum;
```
Comment (DHrytsuk) [inline src/lesson6/CreditCard.java]: What if sum > balance?)

**Comment by NEULOVIM** [comment]
Sorry. I sent the computer twice!

File: src/ADDlesson6/Computer.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
+package ADDlesson6;
+
+public class Computer {
+  private final int price;
```
Comment (DHrytsuk) [inline src/ADDlesson6/Computer.java]: formatting

File: src/ADDlesson6/RAM.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
+package ADDlesson6;
+
+public class RAM {
+   private String name;
+   private int memory;
+
```
Comment (DHrytsuk) [inline src/ADDlesson6/RAM.java]: remove one empty line

File: src/lesson6/CreditCard.java [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+    public CreditCard(int accountNumber, double balance) {
+        this.accountNumber = accountNumber;
+        this.balance = balance;
+    }
+
+
+    public void plusSum(double sum) {
+        balance += sum;
+
```
Comment (DHrytsuk) [inline src/lesson6/CreditCard.java]: remove empty line

File: src/lesson6/CreditCard.java [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    public  double minusSum(double sum) {
+        if (sum > balance) {
+            System.out.println("Error. \n Sum more then balance");
+            return 0;
+        } else {
+            balance -= sum;
+            return sum;
```
Comment (DHrytsuk) [inline src/lesson6/CreditCard.java]: I think, we should return balance

Comment (DHrytsuk) [reply src/lesson6/CreditCard.java]: ?

File: src/ADDlesson6/RAM.java (hunk 5 above)
Comment (DHrytsuk) [reply src/ADDlesson6/RAM.java]: ?

File: src/lesson6/CreditCard.java (hunk 7 above)
Comment (DHrytsuk) [reply src/lesson6/CreditCard.java]: Why this method is stil doesn't return something?

Comment (NEULOVIM) [reply src/lesson6/CreditCard.java]: the idea itself removed the return. I didn't see it, sorry!

**Review by DHrytsuk — APPROVED** [review]

**Pull request merged**